cryptoki/src/mechanism/elliptic_curve.rs

```diff
@@ -93,6 +93,17 @@ impl<'a> EcKdf<'a> {
         }
     }
 
+    /// The sha256 transformation as defined in the x9 standard. The
```

Comment:
```suggestion
    /// The key derivation function based on sha256 as defined in the ANSI X9.63 standard. The
```
I'd recommend this change to make the reference clearer. If you'd like to put in a link to the standard that'd be even nicer 😉